### PR TITLE
Update slides for WarsawJS Workshop #39

### PR DESCRIPTION
**Pull request by lukaszbasaj**
+ update slides

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 node_modules/
+.history/
+.vscode/
```
Comment (malyfko) [inline .gitignore]: these ignores should be configured in your global `.gitignore`, don't add them here

Comment (lukaszbasaj) [reply .gitignore]: Thx. Done.

File: slides.md [hunk 2]
```diff
@@ -49,22 +49,23 @@ class: middle, center, slide-card-list
 
 ]
 
-.card[![](./images/avatars-trainers/dmytro-maretskyi.JPG) Dmytro Maretskyi]
-.card[![](./images/avatars-trainers/chris-trzesniewski.jpeg) Chris Trześniewski]
-.card[![](./images/avatars-trainers/robert-gurgul.jpg) Robert Gurgul]
+.card[![](./images/avatars-trainers/michal-bury.jpg) Michał Bury]
+.card[![](./images/avatars-trainers/robert-kawecki.jpg) Robert Kawecki]
```
[the remark this answers is not in the document]
Comment (lukaszbasaj) [reply slides.md]: Thx. Done.

File: slides.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ class: middle, center, slide-card-list
 ]
 
 .card[![](./images/avatars-volunteers/tatiana-platonova.jpg) Tatiana Platonova]
-.card[![](./images/avatars-volunteers/miroslaw-stasiak.jpg) Mirosław Stasiak]
+<!-- .card[![](./images/avatars-volunteers/miroslaw-stasiak.jpg) Mirosław Stasiak] -->
```
[the remark this answers is not in the document]
Comment (lukaszbasaj) [reply slides.md]: Thx. Done.

**Pull request merged**